pkg/scheduler/apis/config/testing/compatibility_test.go

```diff
@@ -1417,8 +1416,8 @@ func TestAlgorithmProviderCompatibility(t *testing.T) {
 		"PreFilterPlugin": {
 			{Name: "NodeResourcesFit"},
 			{Name: "NodePorts"},
-			{Name: "InterPodAffinity"},
 			{Name: "PodTopologySpread"},
+			{Name: "InterPodAffinity"},
```

Comment:
is the order here important? why did it change?

Reply:
It's necessary. When it was beta, it was **appended** to the last position. And now it's positioned ahead of `InterPodAffinity` - sorted by their cost.